### PR TITLE
Added note about Stacks.js starters interactive CLI

### PR DESCRIPTION
**Pull request by LakshmiLavanyaKasturi**
Relates to #120. 

Added a note to access Stacks.js starters interactive CLI repository. 

Files updated: 

https://docs.hiro.so/build-apps/overview
https://docs.hiro.so/build-apps/authentication
https://docs.hiro.so/build-apps/transaction-signing
https://docs.hiro.so/build-apps/message-signing
https://docs.hiro.so/build-apps/data-storage
https://docs.hiro.so/build-apps/integrate-stacking
https://docs.hiro.so/build-apps/integrate-stacking-delegation

### CODE REVIEW

**Comment by janniks** [comment]
Screen Shot 2022-07-15 at 15 23 37

Not sure this works as intended yet.

Also should we use note/info block to highlight the info and differentiate it from the flow of the page a bit more?

**Comment by janniks** [comment]
> And we don't want a standalone page like ...

Good point! I hadn't seen that. Maybe there's an option in  docusaurus to hide the includes directory..

**Comment by LakshmiLavanyaKasturi** [comment]
> +1 to Jannik's comment ^^. I can't see the included MD showing up correctly in the preview.
> 
> And we don't want a standalone page like https://deploy-preview-121--hiro-docs.netlify.app/includes/stacks.js-starters-note to show up in the path. Perhaps there's a different approach in Docusaurus to include content from other pages as "NOTE".
> 
> Not as a top-level text, but the text being encapsulated inside the NOTE like the following example:
> 
> image

Docusaurus has a different way from the traditional one that I did earlier. Thanks for the link @janniks. The preview seems as expected now. Please check my changes now. @sabbyanandan and @janniks.

File: docs/includes/stacks.js-starters-note.mdx [hunk 1]
```diff
@@ -0,0 +1 @@
+If you're building a frontend application and looking for ways to bootstrap it, you can go to [stacks.js-starters](https://github.com/hirosystems/stacks.js-starters) repo to generate the application with everything automatically assembled for you.
```
Comment (sabbyanandan) [inline docs/includes/stacks.js-starters-note.mdx:1]:
```suggestion
To build a frontend application, you can go to [stacks.js-starters](https://github.com/hirosystems/stacks.js-starters) repo to generate the application with everything automatically assembled for you.
```

**Review by sabbyanandan — CHANGES_REQUESTED** [review]

**Comment by LakshmiLavanyaKasturi** [comment]
@sabbyanandan: Reworded the intro sentence a little bit. Please review! Thanks.

**Review by sabbyanandan — APPROVED** [review]

**Pull request merged**